### PR TITLE
Update install.md with detailed Swift instructions

### PR DESCRIPTION
**Pull request by cnixbtc**
Updates the installation instructions for Swift with detailed examples for Swift Package manager and for CocoaPods.

### CODE REVIEW

File: src/guide/install.md [hunk 1]
```diff
@@ -22,7 +22,39 @@ dependencies {
...
+```
+
+as a package dependency in Xcode.
+
+#### Installation via Swift Package Manifest
+
+Add the following to the dependencies array of your `Package.swift`:
+
+``` swift
+.package(url: "https://github.com/breez/breez-sdk-swift.git", from: "0.0.4"),
```
Comment (roeierez) [inline src/guide/install.md:45]: Can we replace 0.0.4 with latest or similar? Currently the official version is 0.1.0 and it will change constantly.

**Review by roeierez — APPROVED** [review]
Dropped a suggestion for being version agnostic, other than that LGTM!

File: src/guide/install.md (hunk 1 above)
Comment (cnixbtc) [reply src/guide/install.md:45]: There's no such thing as "latest" in the API I fear but maybe we can make this example version agnostic nevertheless by just doing something like: `... from: "<version>")`?

Will update!

**Review by erdemyerebasmaz — APPROVED** [review]

**Comment by ok300** [comment]
@cnixbtc there are some conflicts with `master`, could you please bring this up-to-date?

The PR looks good, would be very cool to merge it!

**Comment by cnixbtc** [comment]
Sure thing, will update tonight!

**Comment by cnixbtc** [comment]
@ok300 conflicts resolved. ready to merge :)

**Pull request merged**